Node/core-DirectLineWebSockets/DirectLineClient/app.js

```diff
@@ -0,0 +1,195 @@
+var Swagger = require('swagger-client');
+var open = require('open');
+var rp = require('request-promise');
+
+// Config settings
+var useW3CWebSocket = false;
+var directLineSecret = 'DIRECTLINE_SECRET';
+var directLineClientName = 'DirectLineClient';
+var directLineSpecUrl = 'https://docs.botframework.com/en-us/restapi/directline3/swagger.json';
+
+process.argv.forEach(function (val, index, array) {
+  if (val === 'w3c') {
```

Comment:
It is the same library, but the API exposed by the two versions change. The NodeJS version uses EventEmitter to trigger events, while the 'w3cwebsocket' follows the W3C spec using event handlers set as props, which is what WebBrowsers do.